tables/de-g0-detailed.utb

```diff
@@ -18,14 +18,14 @@
 #
 # -----------
 #-name: Deutsche Basisschrift
-#-index-name: German, uncontracted, back-translatable
-#-display-name: German uncontracted braille (back-translatable)
+#-index-name: German, uncontracted, detailed
+#-display-name: German uncontracted braille (detailed)
```

Comment:
"Detailed" is maybe a bit cryptic for a display name. I'd have called it something like "German uncontracted braille with indication of capitals" (similar to ru-litbrl-detailed.utb). More precise would be "German uncontracted braille with indication of capitals and detailed representation of accented letters", but that's maybe a bit too long.